addons/binding/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/config/ImageMeta.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pixometer.internal.config;
+
+/**
```

Comment:
It is misplaced it should be before the class not before the imports, so the check is right

Reply:
Gnah. 😄